examples/extract_data/src/extract_data_login.py

```diff
@@ -0,0 +1,136 @@
+LOGIN_URL = 'https://www.glassdoor.com/member/profile'
+IS_LOGGED_IN_PROMPT = "This browser is open to a page that either display's a user's Glassdoor profile or prompts the user to login.  Please give me a JSON response matching the schema below."
+IS_LOGGED_IN_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "isLoggedIn": {
+            "type": "boolean",
+            "description": "Use this field to indicate whether the user is logged in."
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem."
+        },
+    },
+}
+TARGET_URL = 'https://www.glassdoor.com/Job/san-francisco-ca-software-engineer-jobs-SRCH_IL.0,16_IC1147401_KO17,34.htm'
+EXTRACT_DATA_PROMPT = "This browser is open to a page that lists available job roles for software engineers in San Francisco. Please provide 10 job roles that appear to be posted by the AI-related companies."
+EXTRACT_DATA_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "companies": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "companyName": {"type": "string"},
+                    "jobTitle": {"type": "string"},
+                    "location": {"type": "string"},
+                    "salary": {
+                        "type": "object",
+                        "properties": {
+                            "min": {"type": "number", "minimum": 0},
+                            "max": {"type": "number", "minimum": 0},
+                        },
+                        "required": ["min", "max"],
+                    },
+                },
+                "required": ["companyName", "jobTitle", "location", "salary"],
+            },
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem.",
+        },
+    },
+}
+
+async def run():
+    try:
+        if not AIRTOP_API_KEY:
+            raise ValueError("AIRTOP_API_KEY is not set")
+
+        client = Airtop(api_key=AIRTOP_API_KEY)
+
+        profile_id = input("Enter a profileId (or press Enter to skip): ").strip()
+        if profile_id:
+            print(f"Using profileId: {profile_id}")
+        else:
+            print("No profileId provided")
+            profile_id = None
+
+        create_session_response = await client.sessions.create({
+            "configuration": {
+                "timeoutMinutes": 10,
+                "persistProfile": not profile_id,
+                "baseProfileId": profile_id,
+            }
+        })
+
+        session = create_session_response["data"]
+        print("Created airtop session", session["id"])
+
+        if not session.get("cdpWsUrl"):
```

Comment:
This is throwing
<img width="1009" alt="image" src="https://github.com/user-attachments/assets/6e842dc4-55fe-406c-9fc3-6441e5628d14">

This fixes that:
```
        if not session.data.cdp_ws_url:
            raise ValueError("Unable to get cdp url")
```